Bottom: 4701757b3abc3c0a3c730f4716a43140a26260d0
Top:    a4230720d2b5cb882c51131d106e4e7a02f11a69
Author: scott snyder <sss@karma>
Date:   2020-06-10 00:06:05 -0400

Some new strips from gocomics:

adult-children
the-adventures-of-business-cat
aj-and-magnus
alis-house
amanda-the-great
american-chop-suey
angry-birds
ask-a-cat
ask-a-portly-syndicate-person
aunty-acid


---

diff --git a/ComicReader/assets/classes.json b/ComicReader/assets/classes.json
index dc810eb..bfdb7bd 100644
--- a/ComicReader/assets/classes.json
+++ b/ComicReader/assets/classes.json
@@ -108,16 +108,26 @@
     {"class":"GoComics.NinetoFive", "name":"9 to 5", "pref":"9to5Pref"},
     {"class":"GoComics.TheAcademiaWaltz", "name":"The Academia Waltz", "pref":"academiawaltzPref"},
     {"class":"GoComics.AdamatHome", "name":"Adam@Home", "pref":"adamathomePref"},
+    {"class":"GoComics.AdultChildren", "name":"Adult Children", "pref":"adultchildrenPref"},
+    {"class":"GoComics.TheAdventuresOfBusinessCat", "name":"The Adventures of Business Cat", "pref":"theadventuresofbusinesscatPref"},
     {"class":"GoComics.Agnes", "name":"Agnes", "pref":"agnesPref"},
+    {"class":"GoComics.AJAndMagnus", "name":"AJ and Magnus", "pref":"ajandmagnusPref"},
+    {"class":"GoComics.AlisHouse", "name":"Ali's House", "pref":"alishousePref"},
     {"class":"GoComics.AlleyOop", "name":"Alley Oop", "pref":"alleyoopPref"},
+    {"class":"GoComics.AmandaTheGreat", "name":"Amanda the Great", "pref":"amandathegreatPref"},
+    {"class":"GoComics.AmericanChopSuey", "name":"American Chop Suey", "pref":"americanchopsueyPref"},
     {"class":"GoComics.Andertoons", "name":"Andertoons", "pref":"andertoonsPref"},
     {"class":"GoComics.AndyCapp", "name":"Andy Capp", "pref":"andycappPref"},
+    {"class":"GoComics.AngryBirds", "name":"Angry Birds", "pref":"angrybirdsPref"},
     {"class":"GoComics.AngryLittleGirls", "name":"Angry Little Girls", "pref":"angrylittlegirlsPref"},
     {"class":"GoComics.AnimalCrackers", "name":"Animal Crackers", "pref":"animalcrackersPref"},
     {"class":"GoComics.Annie", "name":"Annie", "pref":"anniePref"},
     {"class":"GoComics.TheArgyleSweater", "name":"The Argyle Sweater", "pref":"theargylesweaterPref"},
     {"class":"GoComics.ArloandJanis", "name":"Arlo and Janis", "pref":"arloandjanisPref"},
+    {"class":"GoComics.AskACat", "name":"Ask a Cat", "pref":"askacatPref"},
+    {"class":"GoComics.AskAPortlySyndicatePerson", "name":"Ask a Portly Syndicate Person", "pref":"askaportlysyndicatepersonPref"},
     {"class":"GoComics.AskShagg", "name":"Ask Shagg", "pref":"askshaggPref"},
+    {"class":"GoComics.AuntyAcid", "name":"Aunty Acid", "pref":"auntyacidPref"},
     {"class":"GoComics.BC", "name":"B.C.", "pref":"bcPref"},
     {"class":"GoComics.BackintheDay", "name":"Back in the Day", "pref":"backinthedayPref"},
     {"class":"GoComics.BadReporter", "name":"Bad Reporter", "pref":"badreporterPref"},
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AJAndMagnus.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AJAndMagnus.java
new file mode 100755
index 0000000..eaf16c2
--- /dev/null
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AJAndMagnus.java
@@ -0,0 +1,13 @@
+package com.blogspot.applications4android.comicreader.comics.GoComics;
+
+import java.util.Calendar;
+import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
+
+public class AJAndMagnus extends DailyGoComicsCom {
+	public AJAndMagnus() {
+		super();
+		mComicName = "aj-and-magnus";
+		mFirstCal = Calendar.getInstance();
+		mFirstCal.set(2014, 7, 8);
+	}
+}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AdultChildren.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AdultChildren.java
new file mode 100755
index 0000000..8811557
--- /dev/null
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AdultChildren.java
@@ -0,0 +1,13 @@
+package com.blogspot.applications4android.comicreader.comics.GoComics;
+
+import java.util.Calendar;
+import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
+
+public class AdultChildren extends DailyGoComicsCom {
+	public AdultChildren() {
+		super();
+		mComicName = "adult-children";
+		mFirstCal = Calendar.getInstance();
+		mFirstCal.set(2014, 3, 14);
+	}
+}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AlisHouse.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AlisHouse.java
new file mode 100755
index 0000000..f29d75c
--- /dev/null
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AlisHouse.java
@@ -0,0 +1,13 @@
+package com.blogspot.applications4android.comicreader.comics.GoComics;
+
+import java.util.Calendar;
+import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
+
+public class AlisHouse extends DailyGoComicsCom {
+	public AlisHouse() {
+		super();
+		mComicName = "alis-house";
+		mFirstCal = Calendar.getInstance();
+		mFirstCal.set(2016, 7, 1);
+	}
+}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AmandaTheGreat.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AmandaTheGreat.java
new file mode 100755
index 0000000..891c63b
--- /dev/null
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AmandaTheGreat.java
@@ -0,0 +1,13 @@
+package com.blogspot.applications4android.comicreader.comics.GoComics;
+
+import java.util.Calendar;
+import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
+
+public class AmandaTheGreat extends DailyGoComicsCom {
+	public AmandaTheGreat() {
+		super();
+		mComicName = "amanda-the-great";
+		mFirstCal = Calendar.getInstance();
+		mFirstCal.set(2016, 10, 3);
+	}
+}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AmericanChopSuey.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AmericanChopSuey.java
new file mode 100755
index 0000000..731e799
--- /dev/null
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AmericanChopSuey.java
@@ -0,0 +1,13 @@
+package com.blogspot.applications4android.comicreader.comics.GoComics;
+
+import java.util.Calendar;
+import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
+
+public class AmericanChopSuey extends DailyGoComicsCom {
+	public AmericanChopSuey() {
+		super();
+		mComicName = "american-chop-suey";
+		mFirstCal = Calendar.getInstance();
+		mFirstCal.set(2018, 0, 8);
+	}
+}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AngryBirds.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AngryBirds.java
new file mode 100755
index 0000000..e6b63e7
--- /dev/null
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AngryBirds.java
@@ -0,0 +1,13 @@
+package com.blogspot.applications4android.comicreader.comics.GoComics;
+
+import java.util.Calendar;
+import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
+
+public class AngryBirds extends DailyGoComicsCom {
+	public AngryBirds() {
+		super();
+		mComicName = "angry-birds";
+		mFirstCal = Calendar.getInstance();
+		mFirstCal.set(2017, 7, 3);
+	}
+}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AskACat.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AskACat.java
new file mode 100755
index 0000000..c0d4681
--- /dev/null
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AskACat.java
@@ -0,0 +1,13 @@
+package com.blogspot.applications4android.comicreader.comics.GoComics;
+
+import java.util.Calendar;
+import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
+
+public class AskACat extends DailyGoComicsCom {
+	public AskACat() {
+		super();
+		mComicName = "ask-a-cat";
+		mFirstCal = Calendar.getInstance();
+		mFirstCal.set(2015, 5, 22);
+	}
+}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AskAPortlySyndicatePerson.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AskAPortlySyndicatePerson.java
new file mode 100755
index 0000000..070da79
--- /dev/null
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AskAPortlySyndicatePerson.java
@@ -0,0 +1,13 @@
+package com.blogspot.applications4android.comicreader.comics.GoComics;
+
+import java.util.Calendar;
+import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
+
+public class AskAPortlySyndicatePerson extends DailyGoComicsCom {
+	public AskAPortlySyndicatePerson() {
+		super();
+		mComicName = "ask-a-portly-syndicate-person";
+		mFirstCal = Calendar.getInstance();
+		mFirstCal.set(2019, 6, 8);
+	}
+}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AuntyAcid.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AuntyAcid.java
new file mode 100755
index 0000000..7b9b531
--- /dev/null
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/AuntyAcid.java
@@ -0,0 +1,13 @@
+package com.blogspot.applications4android.comicreader.comics.GoComics;
+
+import java.util.Calendar;
+import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
+
+public class AuntyAcid extends DailyGoComicsCom {
+	public AuntyAcid() {
+		super();
+		mComicName = "aunty-acid";
+		mFirstCal = Calendar.getInstance();
+		mFirstCal.set(2013, 4, 6);
+	}
+}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/TheAdventuresOfBusinessCat.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/TheAdventuresOfBusinessCat.java
new file mode 100755
index 0000000..31d1ac0
--- /dev/null
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/TheAdventuresOfBusinessCat.java
@@ -0,0 +1,13 @@
+package com.blogspot.applications4android.comicreader.comics.GoComics;
+
+import java.util.Calendar;
+import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
+
+public class TheAdventuresOfBusinessCat extends DailyGoComicsCom {
+	public TheAdventuresOfBusinessCat() {
+		super();
+		mComicName = "the-adventures-of-business-cat";
+		mFirstCal = Calendar.getInstance();
+		mFirstCal.set(2015, 3, 27);
+	}
+}
